functions.php

```diff
@@ -4,10 +4,27 @@
  * Additional code for the child theme goes in here.
  */
 
-add_action( 'wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_head', 'hook_javascript');
 
-function enqueue_child_styles() {
+function enqueue_child_styles()
+{
```

Comment:
This really is just stylistic, but I think the [PHP code style guide](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#brace-style) suggests to put brackets on the same line, like this -

```suggestion
function enqueue_child_styles() {
```